src/layouts/UseCases.tsx

```diff
@@ -1,3 +1,448 @@
+      <Show above="lg">
+        <BannerNotification shouldShow zIndex="sticky">
+          <Emoji text=":pencil:" fontSize="2xl" mr={4} flexShrink={0} />
+          <Text m={0}>
+            Uses of Ethereum are always developing and evolving. Add any info
+            you think will make things clearer or more up to date. Edit page
+            {/* TODO: Re-enable after intl implemented */}
+            {/* <Translation id="template-usecase-banner" />{" "}
+            <InlineLink to={absoluteEditPath}>
+              <Translation id="template-usecase-edit-link" />
+            </InlineLink> */}
+          </Text>
+        </BannerNotification>
+      </Show>
+      <HeroContainer>
+        <TitleCard>
+          <Emoji fontSize="4rem" text={frontmatter.emoji!} />
+          <Title>{frontmatter.title}</Title>
+          <Box>
+            <UnorderedList ms="1.45rem">
+              {summaryPoints.map((point, idx) => (
+                <ListItem key={idx} color="text300" mb={0}>
+                  {point}
+                </ListItem>
+              ))}
+            </UnorderedList>
+            <TableOfContents items={tocItems} maxDepth={2} isMobile />
+          </Box>
+        </TitleCard>
+        <Image
+          position="absolute"
+          alignSelf={{ base: "center", lg: "normal" }}
+          top={0}
+          bottom={0}
+          objectFit="cover"
+          width={1000}
+          height={610}
+          src={frontmatter.image}
+          alt={frontmatter.alt || ""}
+          maxW={{
+            base: useCase === "defi" ? "full" : "min(405px, 98%)",
+            lg:
+              (useCase === "dao" && "572px") ||
+              (useCase === "defi" && "80%") ||
+              "640px",
+          }}
+        />
+      </HeroContainer>
+      <Show above="lg">
+        <Flex
+          as={BaseLink}
+          to="#content"
+          bg="ednBackground"
+          justifyContent="center"
+          p={4}
+          width="full"
+          _hover={{
+            bg: "background.base",
+          }}
+        >
+          <Icon as={MdExpandMore} fontSize="2xl" color="secondary" />
+        </Flex>
+      </Show>
+      <Page dir={isRightToLeft ? "rtl" : "ltr"}>
+        {/* <PageMetadata
+          title={frontmatter.title}
+          description={frontmatter.description}
+        /> */}
+        <Show above="lg">
+          <InfoColumn>
+            <StyledButtonDropdown list={dropdownLinks} />
+            <InfoTitle>{frontmatter.title}</InfoTitle>
+
+            {tocItems && (
+              <UpgradeTableOfContents items={tocItems} maxDepth={2} />
+            )}
+          </InfoColumn>
+        </Show>
+        <ContentContainer id="content">
+          {children}
+          <FeedbackCard />
+        </ContentContainer>
+        <Hide above="lg">
+          <MobileButton>
+            <MobileButtonDropdown list={dropdownLinks} />
+          </MobileButton>
+        </Hide>
+      </Page>
+    </Box>
+  )
+}
```

Comment:
Are there any significant changes you remember doing in this file? 

Asking just to keep track of those changes and keep them in mind for future repo syncs.

Reply:
Main thing would probably be adjustments to the HeroContainer and the Image contained within when converting from GatsbyImage